src/hera/toleration.py

```diff
@@ -25,7 +27,7 @@ class Toleration(BaseModel):
     key: str
     effect: str
     operator: str
-    value: str
+    value: Optional[str] = ""
```

Comment:
First, thank you for submitting this PR! Now that you mentioned it as a K8S characteristic, I recalled it's possible to have taints on nodes without a requirement for a value 🙂  What do you think about this:
```suggestion
    value: Optional[str] = None
```
This is more in line with the default values of the `argo-workflows` SDK, which uses `None` pretty much everywhere.

Reply:
updated with None as defaul value